Makefile

```diff
@@ -109,6 +109,9 @@ $(DEPLOY_CONTAINER_MARKER): Dockerfile build-containerized fetch-cni-bins
 .PHONY: fetch-cni-bins
 fetch-cni-bins: dist/flannel dist/loopback dist/host-local
 
+dist/echo-ipam:
```

Comment:
`dist` isn't checked in at the moment.  I could do that if so desired though.  LMK.